### PR TITLE
EUI 2137 link new window

### PR DESCRIPTION
**Pull request by RiteshHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-2137

### Change description ###
Add informative text regarding link to open in new window

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
     "ng": "ng",
     "start:ng": "ng serve --port 3000 --host 0.0.0.0 --proxy-config proxy.config.json",
     "build:ng": "ng build",
-    "test:a11y": "webdriver-manager update --versions.chrome 2.40 && ng e2e --webdriver-update=false --protractorConfig=./test/accessibility/config/protractor.conf.js",
+    "test:a11y": "webdriver-manager update --gecko=false --versions.chrome 2.40 && ng e2e --webdriver-update=false --protractorConfig=./test/accessibility/config/protractor.conf.js",
```
Comment (phillip-whitaker) [inline package.json:36]: why is this needed?

Comment (RiteshHMCTS) [reply package.json:36]: this is the solution Vamshi has given to avoid build failure

**Review by paul-graham — APPROVED** [review]
Pretty simple change. Looks like it does exactly what it says on the tin. I'd _personally_ prefer it didn't target `"_blank"` but that's outside the scope of the ticket. :)

**Review by phillip-whitaker — APPROVED** [review]
